service/src/test/java/eu/europa/ec/fisheries/uvms/activity/service/bean/FluxMessageServiceIT.java

```diff
@@ -37,6 +40,7 @@ public void shouldSaveFaReportDocument() throws Exception {
         JAXBContext jaxbContext = JAXBContext.newInstance(FLUXFAReportMessage.class);
         Unmarshaller jaxbUnmarshaller = jaxbContext.createUnmarshaller();
         FLUXFAReportMessage fluxfaReportMessage = (FLUXFAReportMessage) jaxbUnmarshaller.unmarshal(is);
-        fluxResponseMessageService.saveFishingActivityReportDocuments(fluxfaReportMessage, FaReportSourceEnum.FLUX);
+        FluxFaReportMessageEntity savedEntity = fluxResponseMessageService.saveFishingActivityReportDocuments(fluxfaReportMessage, FaReportSourceEnum.FLUX);
```

Comment:
I know this test isn't even run at the moment, but this quick fix makes Sonar happy :)